sharedbuffers/mapped_struct.py

```diff
@@ -2059,33 +2064,451 @@ def hinted_bsearch(a, hkey, hint):
                 raise NotImplementedError("Unsupported array type %s" % (chr(dtype),))
         finally:
             PyBuffer_Release(cython.address(indexbuf)) #lint:ok
+else:
+    import bisect
+    def _hinted_bsearch(a, hkey, hint):
+        return bisect.bisect_left(a, hkey)
+    globals()['hinted_bsearch'] = _hinted_bsearch
+
+#@cython.ccall
```

Comment:
Should this commented decorators be removed or they are used when running tests or something like that?